cluster/manifests/cluster-lifecycle-controller/rbac.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cluster-lifecycle-controller
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cluster-lifecycle-controller
+rules:
+- apiGroups: [""]
```

Comment:
It doesn't need to create or update pods, so I'd split this into two. It also doesn't need to delete nodes (yet).

Reply:
I changed it so it can patch and delete pods and only patch and update nodes.

Reply:
But why does it need to patch the pods?